gdx/test/com/badlogic/gdx/utils/QueueTest.java

```diff
@@ -290,6 +313,6 @@ private void assertEqualsAndHash (Queue<?> q1, Queue<?> q2) {
 
 	private void assertValues (Queue<Integer> q, Integer... values) {
 		for (int i = 0, n = values.length; i < n; i++)
-			if (values[i] != q.get(i)) fail(q + " != " + new Array(values));
+			if (values[i] == null ? q.get(i) != null : !values[i].equals(q.get(i))) fail(q + " != " + new Array(values));
```

Comment:
This is IntelliJ autogenerated null-safe equals check. I can split it if needed, but as it is just in Test, I didn't deem that necessary.